neuralg/__init__.py

```diff
@@ -1,6 +1,12 @@
 import os
 from loguru import logger
 
+__version__ = "0.0.9"  # TestPyPi index at the moment
+
+from .utils.set_log_level import set_log_level
+
+# Set log level
+set_log_level("INFO")
```

Comment:
Not sure how intense your logging is but generally warning is a better default level to not spam people 🤔